DependencyInjection/Configuration.php

```diff
@@ -23,6 +23,7 @@ public function getConfigTreeBuilder()
 
         $rootNode
             ->children()
+                ->scalarNode('default_formatter')->end() // TODO: make it required when the major version is changed
```

Comment:
an we validate the given input here, as we know which formatters are available, right?

```ifInArray()```